system/mauveLoadTest/playlist.xml

```diff
@@ -38,11 +38,11 @@
 		<platformRequirements>^os.osx</platformRequirements>
 	</test>
 	<test>
-		<testCaseName>MauveSingleThreadLoadTest_HotSpot</testCaseName>
+		<testCaseName>MauveMultiThrdLoad_HS</testCaseName>
 		<variations>
 			<variation>NoOptions</variation>
 		</variations>
-		<command>$(SYSTEMTEST_CMD_TEMPLATE) -test=MauveSingleThreadLoadTest; \
+		<command>$(SYSTEMTEST_CMD_TEMPLATE) -test=MauveMultiThrdLoad; \
```

Comment:
Should this be MauveSingleThreadLoadTest? Not multiple

Reply:
Yes! Corrected.